Assets/Scripts/NarrationItem.cs

```diff
@@ -1,23 +1,38 @@
 using System;
-
+using System.Collections.Generic;
+using System.ComponentModel;
 using UnityEngine;
-[Serializable]
-public class NarrationItem {
-    [SerializeField]
-    private string id;
-    [SerializeField]
-    private string story;
-    [SerializeField]
-    private string section;
-    [SerializeField]
-    private string triggered;
-    [SerializeField]
-    private string character;
-    [SerializeField]
-    private string line;
-    [SerializeField]
-    private string portrait;
-    [SerializeField]
-    private string context;
+using UnityEngine.UI;
+
+[CreateAssetMenu(fileName = "Narrative", menuName = "Narrative/Narrative Item")]
+public class NarrationItem : ScriptableObject{
+    [Tooltip("Day the narration occurs")]
+    public Day                 day;
+    [Tooltip("Character who is speaking")]
+    public Character character;
+    [Tooltip("Sounds that play in order from the beginning of the narration")]
+    public List<AudioClip> sounds;
```

Comment:
I thought that the user would have to read whats in the text area. Instead of having it narrated to them.

Reply:
Yeah, the audio clips would be like little sounds they make. If we implement that. Think animal crossing.